JSTests/wasm/stress/shared-memory-grow-non-crash.js

```diff
@@ -0,0 +1,4 @@
+try {
+    new WebAssembly.Memory({initial : 0, maximum : 1, shared : true}).grow(1)
+} catch {
+}
```

Comment:
The `try-catch` block can be removed. I guess `cvise` was just not smart enough to remove it in the end.

Reply:
@mikhailramalho Since ARM throws an error, we need to keep this try-catch to make every JSC tests pass on all platforms.